db/migrations/00006_create_eth_transaction_cids_table.sql

```diff
@@ -10,6 +10,7 @@ CREATE TABLE eth.transaction_cids (
   src                   VARCHAR(66) NOT NULL,
   tx_data               BYTEA,
   tx_type               BYTEA,
+  gas                   INTEGER NOT NULL,
```

Comment:
Let's call this gas_limit, it's more accurate to how things are defined in EIPs/specs and more explicit about what is is/how it is used.